spec/bike_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rspec'
+require_relative '../lib/dependencies'
+
+RSpec.describe Bike do 
+  before { DeliveryService.new(1,1) }
+  
+  describe '.all method' do    
+    subject { Bike.all }
+    it 'should return all bikes from park' do      
+      expect(subject).to eq(2)
+    end
+  end
+
+  describe '.find_by_available method' do
```

Comment:
provide pool of vehicles and show that ur method returns needful one
for all such tests make same, please

Reply:
Done